packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -153,6 +154,62 @@ export default {
       type: Object,
       default: null,
     },
+
+    /**
+     * Whether the input allows for block quote.
+     */
+    allowBlockquote: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for bold to be introduced in the text.
+     */
+    allowBold: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for bullet list to be introduced in the text.
+    */
+    allowBulletList: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
Error when enabling this in storybook.

![Screenshot 2024-04-09 at 1 05 07 PM](https://github.com/dialpad/dialtone/assets/64808812/66ff0661-acd8-43a6-862b-98a6967dec7b)

Reply:
should be fixed now as I removed that prop. FYI I created a ticket some time ago because it seems that the list is not working: https://dialpad.atlassian.net/browse/DLT-1668

Reply:
ah okay thanks.. We'll look at that bug